.evergreen/config.yml

```diff
@@ -1675,6 +1689,14 @@ buildvariants:
   tasks:
     - "compile-only"
 
+- matrix_name: "cargo-deny"
```

Comment:
Since we don't need any special matrix configuration, would it be fine to just include a new task in the `Lint` matrix?

Reply:
Good call, done.